app/assets/stylesheets/modules/_selects.styl

```diff
@@ -40,6 +40,7 @@
     background url('../images/sort.svg') no-repeat 48% white
     -webkit-appearance none
     text-indent -200px
+    margin-top -10px
```

Comment:
This is going to affect every sort-select, not just the one in the screenshot.

I think this adjustment either needs to be done in a way that just affects that uploads page, or in a way that address the same issue for every view that includes a sort-select without any code specific to uploads.

Reply:
@ragesoss can I create a new className element for just the uploads page and make styling changes in that?

Reply:
No, that will lead to more styling trouble later one, if/when we try to adjust the styles of those selects and it leaves just one of them unchanged. Doing it in a way that would be beneficial for all the sort-selects is much preferable to a fix that just deals with this one.

Reply:
I had a look at most of the sort-select buttons and it's only in the mobile view that there are layout issues. So I'll add **margin-top -10px** just to the mobile version.
Is that fine @ragesoss ?

Reply:
Applying that code will not create any disorientation in other pages that have sort-select

Reply:
Okay. Can you include screenshots of before / after for both the uploads page and one of the other pages with a sort-select?